src/store/actions/api.actions.js

```diff
@@ -398,123 +392,62 @@ export const deleteEndpoint = apiName => dispatch => {
-            dispatch(openResponseModal({
-                status: error.response.status,
-                statusText: error.response.statusText,
-                message: error.response.data,
-            }));
-
-        // The request was made and the server responded with a status code
-        // that falls out of the range of 2xx
-        // More info about error handling in Axios: https://github.com/mzabriskie/axios#handling-errors
-            // eslint-disable-next-line
-            console.error(error.response.data);
-        } else if (error.request) {
-        // The request was made but no response was received
-        // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
-        // http.ClientRequest in node.js
-            // eslint-disable-next-line
-            console.log(error.request);
-        } else {
-        // Something happened in setting up the request that triggered an Error
-            // eslint-disable-next-line
-            console.log('Error', error.message);
-        }
     }
 };
 
-export const confirmedUpdateEndpoint = (dispatch, pathname, api) => {
+export const confirmedUpdateEndpoint = async (dispatch, pathname, api) => {
     dispatch(saveEndpointRequest());
+    dispatch(closeConfirmationModal());
 
     const preparedPlugins = preparePlugins(api);
     // substitude updated list of plugins
     const preparedApi = R.set(R.lensPath(['plugins']), preparedPlugins, api);
 
-    return client.put(`apis${pathname}`, preparedApi)
-        .then((response) => {
-            dispatch(saveEndpointSuccess());
-            dispatch(closeConfirmationModal());
-            dispatch(showToaster());
-        })
-        .catch((error) => {
-            if (error.response) {
-                dispatch(openResponseModal({
-                    status: error.response.status,
-                    statusText: error.response.statusText,
-                    message: error.response.data,
-                }));
-
-            // The request was made and the server responded with a status code
-            // that falls out of the range of 2xx
-            // More info about error handling in Axios: https://github.com/mzabriskie/axios#handling-errors
-                // eslint-disable-next-line
-                console.error(error.response.data);
-            } else if (error.request) {
-            // The request was made but no response was received
-            // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
-            // http.ClientRequest in node.js
-                // eslint-disable-next-line
-                console.log(error.request);
-            } else {
-            // Something happened in setting up the request that triggered an Error
-                // eslint-disable-next-line
-                console.log('Error', error.message);
-            }
-        });
+    try {
+        const response = await client.put(`apis${pathname}`, preparedApi);
```

Comment:
response is useless in this case right? Having `await client.put(`apis${pathname}`, preparedApi);` only would do the trick

Reply:
`true`